lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,33 @@
+package DDG::Spice::IndegoBikeshareLocations;
+
+use strict;
+use DDG::Spice;
+
+name 'PHL Indego Bikeshare Locations';
+source 'Indego Philadelphia';
+icon_url 'http://www.rideindego.com/';
+description 'Search Indego Bike stations in Philadelphia';
+primary_example_queries 'indego philadelphia', 'bikeshare philadelphia';
+secondary_example_queries 'philly bikeshare';
+category 'geography';
+topics 'everyday', 'travel', 'entertainment', 'geography';
+code_url 'https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/PHLBikeshare/IndegoBikeshareLocations.pm';
+attribution github => 'marianosimone',
+            github => 'AcriCAA', 
+            web => ['http://www.marianosimone.com',  'Mariano Simone'],
+            web  => ['http://www.coreyacri.com',  'Corey Acri'];
+
+triggers any => 'indego', 'bikeshare', 'bike share', 'ride indego';
+
+spice to => 'https://api.phila.gov/bike-share-stations/v1';
+spice wrap_jsonp_callback => 1;
+spice proxy_cache_valid => '200 304 15m';
```

Comment:
Thanks, @marianosimone.

@jagtalon, would you prefer I delete that whole line then?

On Tue, Aug 18, 2015 at 2:44 PM, Mariano Simone <notifications@github.com>
wrote:

> In lib/DDG/Spice/IndegoBikeshareLocations.pm
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102#discussion_r37337143>
> :
>
> > +description 'Search Indego Bike stations in Philadelphia';
> > +primary_example_queries 'indego philadelphia', 'bikeshare philadelphia';
> > +secondary_example_queries 'philly bikeshare';
> > +category 'geography';
> > +topics 'everyday', 'travel', 'entertainment', 'geography';
> > +code_url 'https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/PHLBikeshare/IndegoBikeshareLocations.pm';
> > +attribution github => 'marianosimone',
> > +            github => 'AcriCAA',
> > +            web => ['http://www.marianosimone.com',  'Mariano Simone'],
> > +            web  => ['http://www.coreyacri.com',  'Corey Acri'];
> > +
> > +triggers any => 'indego', 'bikeshare', 'bike share', 'ride indego';
> > +
> > +spice to => 'https://api.phila.gov/bike-share-stations/v1';
> > +spice wrap_jsonp_callback => 1;
> > +spice proxy_cache_valid => '200 304 15m';
>
> @jagtalon <https://github.com/jagtalon> @AcriCAA
> <https://github.com/AcriCAA> yeag... I actually thought it would be a
> good idea at first... but as we are showing availability, we might NOT want
> to cache
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102/files#r37337143>
> .
>